src/containers/facility/FacilityIntroduction.tsx

```diff
@@ -103,11 +108,11 @@ const HotelAddress = ({ address }: { address?: string }) => {
 };
 
 const HeaderTitle = ({ name, address }: { name?: string; address?: string }) => {
+  const theme = useTheme();
   return (
     <HeaderTitleContainer>
-      <Button sx={{ marginRight: '8px' }}>{'<'}</Button>
       <div>
-        <Typography fontWeight={500} fontSize="2rem" marginBottom={'10px'}>
+        <Typography variant="h1" marginBottom={theme.spacing(1.5)}>
```

Comment:
I believe you can also just pass 1.5 as a number and it will automatically use the sizing the the `theme`.